assets/js/components.js

```diff
@@ -57,6 +57,19 @@ function accordion($el) {
   return new Accordion($el);
 }
 
+function togglePassword($el) {
+  var fieldSelector =  '#' + $el.attr('aria-controls'),
```

Comment:
+1 gotcha! I missed where the `aria-controls` were located and was not entirely sure how the accessibility portion was integrated in. Thank you for clarifying!